modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/util/ExecutionUtils.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.module.extension.internal.util;
+
+import org.mule.runtime.extension.api.runtime.config.ConfigurationInstance;
+import org.mule.runtime.extension.api.runtime.operation.ExecutionContext;
+import org.mule.runtime.module.extension.internal.runtime.config.MutableConfigurationStats;
+
+import java.util.Optional;
+
+public class ExecutionUtils {
+
+  public static Optional<MutableConfigurationStats> getMutableConfigurationStats(ExecutionContext<?> context) {
```

Comment:
move this method to `org.mule.runtime.module.extension.internal.util.MuleExtensionUtils` and delete this class. Also, add jdocs (don't forget the `@since`)